main/auth/courses_categories.php

```diff
@@ -329,17 +329,17 @@ function return_teacher($course)
     $length = count($teachers);
     foreach ($teachers as $value) {
         $name = $value['firstname'].' ' . $value['lastname'];
+
         if ($length > 2) {
              $html .= '<a href="'.$value['url'].'" class="ajax" data-title="'.$name.'">
-                    <img src="'.$value['avatar'].'"/></a>';
+                    <img src="'.$value['avatar'].'" alt="'.$name.' ' .get_lang('Profile').'"/></a>';
```

Comment:
Usually we try to avoid (now - many cases have been inserted in the past) to have translated strings mixed with variables in a way in which the translated string cannot "include" the variable, because in RTL languages, the order of things might not be understandable as we have them (that's not only true for RTL languages). See our wiki page on languages management https://github.com/chamilo/chamilo-lms/wiki/Coding-conventions#translations-management
In any case, this isn't a big issue, but I wanted to make sure you knew the rules.